reverseArray.js

```diff
@@ -0,0 +1,20 @@
+function transformArrayOfNumbers(array) {
+    //fliter and return odd numbers
+   const oddNumbers =  array.filter( (arrNumber)=> {
```

Comment:
Good job with the use of `filter` method @Oyinlolatechie. A simpler way to write this without needing an extra `return` statement would be:

```
const oddNumbers = array.filter((arrNumber) => arrNumber % 2 === 1)
```